app/pages/UcsfClientEligibilityPage/ucsfEligibilities.ts

```diff
@@ -0,0 +1,101 @@
+// We are hardcoding the eligibility data as it currently exists on the design specs.
+// When content has entered UCSF specific eligibilities, will we fetch that data from the API
+
+export interface ucsfEligibilityDictionary {
+  [key: string]: object;
```

Comment:
I think it would be valuable to give this a more precise type, since we know exactly the shape of the data we expect here, and adding a more precise type will allow TypeScript to enforce better type safety everywhere else in the app. Based on the data below, I think the following would be the right type definition:

```ts
interface Eligibility {
  name: string;
  checked: boolean;
}

interface EligibilityGroup {
  label: string;
  eligibilities: Eligibility[];
}

export interface UcsfEligibilityDictionary {
  [key: string]: EligibilityGroup[];
}
```

Reply:
Great, this also fixes an issue with type inference that you called out above. Thanks!